cfgov/wagtailadmin_overrides/locale/es/LC_MESSAGES/django.po

```diff
@@ -0,0 +1,101 @@
+# SOME DESCRIPTIVE TITLE.
```

Comment:
Could we exclude wagtailadmin_overrides from this, in the interest of simplification? This app was intended only to provide our custom overrides for the Wagtail admin itself, where I can't imagine we'd ever want to override their translations. I'm pretty sure that the installation of the Wagtail package itself pulls in their translations, so we shouldn't need to provide these.

Additionally, with the exception of [this one CSS file](https://github.com/cfpb/consumerfinance.gov/blob/main/cfgov/wagtailadmin_overrides/static/css/form-explainer.css), the only functionality this app provides is page descriptions (added in #7134), which we backported from Wagtail 3.0. If we're planning on upgrading soon, 99% of this app will be going away.

Reply:
Well. We can, but AFAIK it means we'd have to specify that exclusion as [an argument to `makemessages` every time any of us runs it](https://docs.djangoproject.com/en/3.2/ref/django-admin/#django-admin-makemessages). Otherwise it's going to pick up gettext patterns there.

I'd be okay with that, but we'd be relying on ourselves to always use that `--exclude` when we run `makemessages`, otherwise we'll get those messages added to our `.po` files.

Reply:
An alternative might be using our custom `makemessages` to add `wagtailadmin_overrides` to the default ignore list.

Reply:
Great point, okay. In that case I wonder/hope that when we remove those override templates upon an upgrade to 3.0, there won't be any translatable text in the app, and we can remove those files then.

Reply:
It's worth noting too, if there's no `msgstr`, I don't think it ends up in the compiled messages file at all, so in effect these would be noops. I'm experimenting to confirm that.